examples/advanced/lr-newton-raphson/README.md

```diff
@@ -0,0 +1,243 @@
+# Federated Logistic Regression with Second-Order Newton-Raphson optimization
+
+This example shows how to implement a federated binary
+classification via logistic regression with second-order Newton-Raphson optimization.
+
+The [UCI Heart Disease
+dataset](https://archive.ics.uci.edu/dataset/45/heart+disease) is
+used in this example. Scripts are provided to download and process the
+dataset as described
+[here](https://github.com/owkin/FLamby/tree/main/flamby/datasets/fed_heart_disease). This
+dataset contains samples from 4 sites, splitted into training and
+testing sets as described below:
+|site         | sample split                          |
+|-------------|---------------------------------------|
+|Cleveland    | train: 199 samples, test: 104 samples |
+|hungary      | train: 172 samples, test: 89 samples  |
+|Switzerland  | train: 30 samples, test: 16 samples   |
+|Long Beach V | train: 85 samples, test: 45 samples   |
+
+The number of features in each sample is 13.
+
+## Introduction
+
+The Newton-Raphson optimization problem can be described as follows.
```

Comment:
Maybe add a link to wikipedia https://en.wikipedia.org/wiki/Newton%27s_method

Reply:
Added the link to wikipedia